console.log("start");

try {
    unknownFunction();
} catch (error) {
    console.log("fix this error");
}

console.log("end");
